scripts/publish.sh

```diff
@@ -46,7 +46,7 @@ eu-west-3
 eu-north-1
 "
 
-classic_regions_account_id="906394416424"
+classic_regions_account_id="075490442118"
```

Comment:
Did we create a new account?

Reply:
No, this is my account.. for testing purpose. Will modify this part after the test.

Reply:
Ahh kind of confused. Can't we test before merging the code into mainline?

Reply:
Yes, we can. But for this PR, I didn't mean to merge it now. This is not the final version for it and this PR is just a way for members to leave comments. Sorry for the confusion..